Source/Hurl.BrowserSelector/Controls/BrowsersList.xaml

```diff
@@ -45,60 +45,40 @@
 
         <ItemsControl.ItemTemplate>
             <DataTemplate>
-                <Border
-                    x:Name="BtnArea"
-                    Width="120"
-                    MaxHeight="130"
-                    Margin="2"
-                    Padding="5,10,5,10"
-                    BorderThickness="0"
-                    CornerRadius="5"
-                    Cursor="Hand"
-                    Focusable="True"
-                    MouseLeftButtonUp="BrowserButton_Click"
+                <wpfui:Button
+                    Width="128"
+                    Margin="4"
+                    VerticalAlignment="Stretch"
+                    VerticalContentAlignment="Top"
+                    Appearance="Transparent"
+                    AutomationProperties.Name="{Binding Path=Name}"
```

Comment:
Does this property do anything here ?

Reply:
Yes, it's for the Screen Reader. When receiving Tab focus, it calls out the name of the browser. Otherwise, there is no (audio) feedback.